[WARNING] 2025-07-17 09:08:26,301 request_logger (38): GET / → 404
[WARNING] 2025-07-17 09:08:26,302 django.request (248): Not Found: /
[WARNING] 2025-07-17 09:08:26,303 django.server (213): "GET / HTTP/1.1" 404 2821
[WARNING] 2025-07-17 09:08:26,415 request_logger (38): GET /favicon.ico → 404
[WARNING] 2025-07-17 09:08:26,417 django.request (248): Not Found: /favicon.ico
[WARNING] 2025-07-17 16:21:36,929 request_logger (38): GET / → 404
[WARNING] 2025-07-17 16:21:36,931 django.request (248): Not Found: /
[WARNING] 2025-07-17 16:21:36,932 django.server (213): "GET / HTTP/1.1" 404 2821
[WARNING] 2025-07-17 16:21:37,100 request_logger (38): GET /favicon.ico → 404
[WARNING] 2025-07-17 16:21:37,104 django.request (248): Not Found: /favicon.ico
[WARNING] 2025-07-17 16:37:48,045 request_logger (38): POST /api/v1/user/register/ → 422
[WARNING] 2025-07-17 16:37:48,060 django.request (248): Unprocessable Entity: /api/v1/user/register/
[WARNING] 2025-07-17 16:37:48,061 django.server (213): "POST /api/v1/user/register/ HTTP/1.1" 422 111
[WARNING] 2025-07-17 16:37:55,241 request_logger (38): POST /api/v1/user/register/ → 422
[WARNING] 2025-07-17 16:37:55,242 django.request (248): Unprocessable Entity: /api/v1/user/register/
[WARNING] 2025-07-17 16:37:55,243 django.server (213): "POST /api/v1/user/register/ HTTP/1.1" 422 111
[WARNING] 2025-07-17 16:39:45,134 request_logger (38): POST /api/v1/user/register/ → 422
[WARNING] 2025-07-17 16:39:45,135 django.request (248): Unprocessable Entity: /api/v1/user/register/
[WARNING] 2025-07-17 16:39:45,136 django.server (213): "POST /api/v1/user/register/ HTTP/1.1" 422 93
[WARNING] 2025-07-17 17:12:23,338 request_logger (38): POST /api/v1/user/register/ → 422
[WARNING] 2025-07-17 17:12:23,339 django.request (248): Unprocessable Entity: /api/v1/user/register/
[WARNING] 2025-07-17 17:12:23,340 django.server (213): "POST /api/v1/user/register/ HTTP/1.1" 422 111
[WARNING] 2025-07-17 19:14:28,679 request_logger (38): POST /api/v1/user/register/ → 422
[WARNING] 2025-07-17 19:14:28,680 django.request (248): Unprocessable Entity: /api/v1/user/register/
[WARNING] 2025-07-17 19:14:28,681 django.server (213): "POST /api/v1/user/register/ HTTP/1.1" 422 168
[WARNING] 2025-07-17 19:49:40,407 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 19:49:40,408 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 19:49:40,409 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 164
[WARNING] 2025-07-17 19:50:14,813 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 19:50:14,814 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 19:50:14,815 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 164
[WARNING] 2025-07-17 19:50:57,158 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 19:50:57,161 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 19:50:57,162 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 107
[WARNING] 2025-07-17 20:11:40,577 request_logger (38): POST /api/v1/timeslot/create/ → 404
[WARNING] 2025-07-17 20:11:40,578 django.request (248): Not Found: /api/v1/timeslot/create/
[WARNING] 2025-07-17 20:11:40,579 django.server (213): "POST /api/v1/timeslot/create/ HTTP/1.1" 404 15841
[WARNING] 2025-07-17 20:14:22,334 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:14:22,335 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:14:22,335 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 101
[WARNING] 2025-07-17 20:17:54,771 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:17:54,772 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:17:54,773 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:20:51,960 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:20:51,962 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:20:51,966 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:21:02,290 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:21:02,291 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:21:02,292 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:21:12,850 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:21:12,852 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:21:12,853 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:23:49,629 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:23:49,630 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:23:49,631 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 107
[WARNING] 2025-07-17 20:23:56,474 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:23:56,475 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:23:56,476 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:24:52,683 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:24:52,684 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:24:52,685 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:25:02,953 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:25:02,955 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:25:02,955 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:26:49,889 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:26:49,890 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:26:49,891 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:28:42,968 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:28:42,969 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:28:42,971 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:29:06,671 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:29:06,671 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:29:06,672 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:29:11,871 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:29:11,872 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:29:11,873 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:29:15,242 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:29:15,243 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:29:15,244 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:29:20,900 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:29:20,901 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:29:20,901 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-17 20:29:36,111 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-17 20:29:36,112 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-17 20:29:36,113 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 112
[WARNING] 2025-07-18 05:13:06,340 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 05:13:06,358 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 05:13:06,359 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 145
[WARNING] 2025-07-18 05:13:28,298 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 05:13:28,299 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 05:13:28,300 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 145
[WARNING] 2025-07-18 05:13:41,643 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 05:13:41,644 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 05:13:41,645 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 145
[WARNING] 2025-07-18 05:13:57,306 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 05:13:57,307 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 05:13:57,308 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 145
[WARNING] 2025-07-18 07:25:06,154 request_logger (38): GET /api/v1/filter/doctors/?division_id=1 → 404
[WARNING] 2025-07-18 07:25:06,156 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:25:06,157 django.server (213): "GET /api/v1/filter/doctors/?division_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:25:35,456 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:25:35,457 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:25:35,458 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:25:37,921 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:25:37,923 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:25:37,924 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:26:14,049 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:26:14,051 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:26:14,052 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:26:16,061 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:26:16,062 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:26:16,063 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:26:19,416 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:26:19,417 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:26:19,418 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:26:25,375 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:26:25,376 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:26:25,377 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:28:34,212 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:28:34,213 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:28:34,215 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:28:36,605 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:28:36,607 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:28:36,608 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:28:38,397 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:28:38,398 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:28:38,399 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:28:39,858 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:28:39,860 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:28:39,860 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:28:46,879 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:28:46,880 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:28:46,881 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:28:53,311 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 07:28:53,312 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:28:53,312 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:28:59,826 request_logger (38): GET /api/v1/filter/doctors/?division_id=1 → 404
[WARNING] 2025-07-18 07:28:59,827 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:28:59,828 django.server (213): "GET /api/v1/filter/doctors/?division_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:31:10,903 request_logger (38): GET /api/v1/filter/doctors/?division_id=1 → 404
[WARNING] 2025-07-18 07:31:10,904 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:31:10,905 django.server (213): "GET /api/v1/filter/doctors/?division_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:31:49,193 request_logger (38): GET /api/v1/filter/doctors/?division_id=1 → 404
[WARNING] 2025-07-18 07:31:49,194 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:31:49,195 django.server (213): "GET /api/v1/filter/doctors/?division_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:32:47,152 request_logger (38): GET /api/v1/filter/doctors/?division_id=1 → 404
[WARNING] 2025-07-18 07:32:47,153 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:32:47,154 django.server (213): "GET /api/v1/filter/doctors/?division_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:32:49,537 request_logger (38): GET /api/v1/filter/doctors/?division_id=1 → 404
[WARNING] 2025-07-18 07:32:49,538 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:32:49,539 django.server (213): "GET /api/v1/filter/doctors/?division_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:33:11,497 request_logger (38): GET /api/v1/filter/doctors/?division_id=1 → 404
[WARNING] 2025-07-18 07:33:11,498 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:33:11,500 django.server (213): "GET /api/v1/filter/doctors/?division_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:33:13,273 request_logger (38): GET /api/v1/filter/doctors/?division_id=1 → 404
[WARNING] 2025-07-18 07:33:13,274 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:33:13,275 django.server (213): "GET /api/v1/filter/doctors/?division_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:33:54,813 request_logger (38): GET /api/v1/filter/doctors/?division_id=1 → 404
[WARNING] 2025-07-18 07:33:54,814 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:33:54,814 django.server (213): "GET /api/v1/filter/doctors/?division_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:33:57,119 request_logger (38): GET /api/v1/filter/doctors/?division_id=1 → 404
[WARNING] 2025-07-18 07:33:57,120 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:33:57,121 django.server (213): "GET /api/v1/filter/doctors/?division_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:46:02,454 request_logger (38): GET /api/v1/filter/doctors/?thana_id=1 → 404
[WARNING] 2025-07-18 07:46:02,455 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:46:02,456 django.server (213): "GET /api/v1/filter/doctors/?thana_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:48:45,630 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&thana_id=1 → 404
[WARNING] 2025-07-18 07:48:45,632 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:48:45,633 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&thana_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 07:48:56,352 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1&thana_id=1 → 404
[WARNING] 2025-07-18 07:48:56,353 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 07:48:56,354 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1&thana_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 08:03:19,853 request_logger (38): GET /api/v1/filter/doctors/?division_id=1&district_id=1 → 404
[WARNING] 2025-07-18 08:03:19,854 django.request (248): Not Found: /api/v1/filter/doctors/
[WARNING] 2025-07-18 08:03:19,855 django.server (213): "GET /api/v1/filter/doctors/?division_id=1&district_id=1 HTTP/1.1" 404 59
[WARNING] 2025-07-18 12:35:14,291 request_logger (38): GET / → 404
[WARNING] 2025-07-18 12:35:14,292 django.request (248): Not Found: /
[WARNING] 2025-07-18 12:35:14,293 django.server (213): "GET / HTTP/1.1" 404 2821
[WARNING] 2025-07-18 12:35:14,604 request_logger (38): GET /favicon.ico → 404
[WARNING] 2025-07-18 12:35:14,607 django.request (248): Not Found: /favicon.ico
[WARNING] 2025-07-18 18:06:05,655 request_logger (38): POST /api/v1/user/register/ → 422
[WARNING] 2025-07-18 18:06:05,685 django.request (248): Unprocessable Entity: /api/v1/user/register/
[WARNING] 2025-07-18 18:06:05,686 django.server (213): "POST /api/v1/user/register/ HTTP/1.1" 422 83
[WARNING] 2025-07-18 18:12:08,067 request_logger (38): POST /api/v1/time/slot/create/ → 422
[WARNING] 2025-07-18 18:12:08,068 django.request (248): Unprocessable Entity: /api/v1/time/slot/create/
[WARNING] 2025-07-18 18:12:08,069 django.server (213): "POST /api/v1/time/slot/create/ HTTP/1.1" 422 81
[WARNING] 2025-07-18 18:20:15,015 request_logger (38): POST /api/v1/time/slot/create/ → 422
[WARNING] 2025-07-18 18:20:15,016 django.request (248): Unprocessable Entity: /api/v1/time/slot/create/
[WARNING] 2025-07-18 18:20:15,017 django.server (213): "POST /api/v1/time/slot/create/ HTTP/1.1" 422 100
[WARNING] 2025-07-18 18:25:39,926 request_logger (38): POST /api/v1/doctor/profile/create/ → 422
[WARNING] 2025-07-18 18:25:39,927 django.request (248): Unprocessable Entity: /api/v1/doctor/profile/create/
[WARNING] 2025-07-18 18:25:39,928 django.server (213): "POST /api/v1/doctor/profile/create/ HTTP/1.1" 422 101
[WARNING] 2025-07-18 18:25:43,638 request_logger (38): POST /api/v1/doctor/profile/create/ → 422
[WARNING] 2025-07-18 18:25:43,643 django.request (248): Unprocessable Entity: /api/v1/doctor/profile/create/
[WARNING] 2025-07-18 18:25:43,644 django.server (213): "POST /api/v1/doctor/profile/create/ HTTP/1.1" 422 101
[WARNING] 2025-07-18 18:25:45,937 request_logger (38): POST /api/v1/doctor/profile/create/ → 422
[WARNING] 2025-07-18 18:25:45,938 django.request (248): Unprocessable Entity: /api/v1/doctor/profile/create/
[WARNING] 2025-07-18 18:25:45,939 django.server (213): "POST /api/v1/doctor/profile/create/ HTTP/1.1" 422 101
[WARNING] 2025-07-18 18:27:14,138 request_logger (38): POST /api/v1/doctor/profile/create/ → 422
[WARNING] 2025-07-18 18:27:14,139 django.request (248): Unprocessable Entity: /api/v1/doctor/profile/create/
[WARNING] 2025-07-18 18:27:14,140 django.server (213): "POST /api/v1/doctor/profile/create/ HTTP/1.1" 422 98
[WARNING] 2025-07-18 18:27:18,853 request_logger (38): POST /api/v1/doctor/profile/create/ → 422
[WARNING] 2025-07-18 18:27:18,854 django.request (248): Unprocessable Entity: /api/v1/doctor/profile/create/
[WARNING] 2025-07-18 18:27:18,855 django.server (213): "POST /api/v1/doctor/profile/create/ HTTP/1.1" 422 101
[WARNING] 2025-07-18 18:27:23,682 request_logger (38): POST /api/v1/doctor/profile/create/ → 422
[WARNING] 2025-07-18 18:27:23,683 django.request (248): Unprocessable Entity: /api/v1/doctor/profile/create/
[WARNING] 2025-07-18 18:27:23,684 django.server (213): "POST /api/v1/doctor/profile/create/ HTTP/1.1" 422 101
[WARNING] 2025-07-18 18:27:26,944 request_logger (38): POST /api/v1/doctor/profile/create/ → 422
[WARNING] 2025-07-18 18:27:26,945 django.request (248): Unprocessable Entity: /api/v1/doctor/profile/create/
[WARNING] 2025-07-18 18:27:26,946 django.server (213): "POST /api/v1/doctor/profile/create/ HTTP/1.1" 422 101
[WARNING] 2025-07-18 18:27:35,030 request_logger (38): POST /api/v1/doctor/profile/create/ → 422
[WARNING] 2025-07-18 18:27:35,032 django.request (248): Unprocessable Entity: /api/v1/doctor/profile/create/
[WARNING] 2025-07-18 18:27:35,033 django.server (213): "POST /api/v1/doctor/profile/create/ HTTP/1.1" 422 101
[WARNING] 2025-07-18 18:28:05,156 request_logger (38): POST /api/v1/time/slot/create/ → 422
[WARNING] 2025-07-18 18:28:05,157 django.request (248): Unprocessable Entity: /api/v1/time/slot/create/
[WARNING] 2025-07-18 18:28:05,158 django.server (213): "POST /api/v1/time/slot/create/ HTTP/1.1" 422 100
[WARNING] 2025-07-18 18:31:11,478 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:31:11,479 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:31:11,480 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 100
[WARNING] 2025-07-18 18:31:17,985 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:31:17,986 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:31:17,987 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:31:24,725 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:31:24,726 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:31:24,726 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:31:30,090 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:31:30,092 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:31:30,093 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:31:34,631 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:31:34,632 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:31:34,633 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:33:44,295 request_logger (38): POST /api/v1/time/slot/list/ → 405
[WARNING] 2025-07-18 18:33:44,297 django.request (248): Method Not Allowed: /api/v1/time/slot/list/
[WARNING] 2025-07-18 18:33:44,298 django.server (213): "POST /api/v1/time/slot/list/ HTTP/1.1" 405 41
[WARNING] 2025-07-18 18:36:37,648 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:36:37,649 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:36:37,650 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 101
[WARNING] 2025-07-18 18:36:49,820 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:36:49,823 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:36:49,825 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 101
[WARNING] 2025-07-18 18:37:41,347 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:37:41,349 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:37:41,350 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:38:27,726 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:38:27,727 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:38:27,728 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:38:34,438 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:38:34,445 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:38:34,446 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:38:37,771 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:38:37,774 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:38:37,781 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:38:41,952 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:38:41,953 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:38:41,954 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:38:47,140 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:38:47,142 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:38:47,143 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 145
[WARNING] 2025-07-18 18:38:53,243 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:38:53,244 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:38:53,245 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:39:08,761 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:39:08,762 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:39:08,763 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:39:22,886 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:39:22,887 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:39:22,889 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:39:29,216 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:39:29,217 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:39:29,218 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:39:39,519 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:39:39,520 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:39:39,521 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:39:44,277 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:39:44,277 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:39:44,278 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:39:47,526 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:39:47,528 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:39:47,529 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:39:52,647 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:39:52,648 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:39:52,649 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:40:04,195 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:40:04,196 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:40:04,197 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 108
[WARNING] 2025-07-18 18:40:12,220 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:40:12,221 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:40:12,222 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:40:18,795 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:40:18,796 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:40:18,797 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:40:24,662 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:40:24,662 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:40:24,663 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:40:28,328 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:40:28,329 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:40:28,329 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:40:31,602 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:40:31,603 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:40:31,604 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:40:34,967 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:40:34,969 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:40:34,970 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:40:39,988 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:40:39,989 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:40:39,990 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:40:43,799 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:40:43,800 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:40:43,801 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:40:58,604 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:40:58,605 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:40:58,606 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:41:04,264 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:41:04,265 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:41:04,266 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:41:07,329 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:41:07,331 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:41:07,332 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:41:14,035 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:41:14,036 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:41:14,037 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:41:24,862 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:41:24,863 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:41:24,864 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:41:29,686 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:41:29,687 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:41:29,688 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:41:32,444 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:41:32,446 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:41:32,447 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:41:37,850 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:41:37,852 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:41:37,853 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:41:49,581 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:41:49,582 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:41:49,583 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
[WARNING] 2025-07-18 18:41:54,841 request_logger (38): POST /api/v1/appointment/create/ → 422
[WARNING] 2025-07-18 18:41:54,842 django.request (248): Unprocessable Entity: /api/v1/appointment/create/
[WARNING] 2025-07-18 18:41:54,842 django.server (213): "POST /api/v1/appointment/create/ HTTP/1.1" 422 105
